Tests for utils  don't pass

Tests in util_tests.go don't pass.
One of results
```
                Test:           Test_objectInfoFromMeta/a/b/small.jpg
    --- FAIL: Test_objectInfoFromMeta/a/b/c/small.jpg#01 (0.00s)
        util_test.go:151: 
                Error Trace:    util_test.go:151
                Error:          Not equal: 
                                expected: &layer.ObjectInfo{id:(*object.ID)(0xc00013edb0), isDir:false, Bucket:"test-container", Name:"a/b/c/small.jpg", Size:19, ContentType:"text/plain; charset=utf-8", Created:time.Time{wall:0x0, ext:63761267128, loc:(*time.Location)(0x1039880)}, HashSum:"", Owner:(*owner.ID)(0xc00013ed98), Headers:map[string]string{}}
                                actual  : &layer.ObjectInfo{id:(*object.ID)(0xc00013edb0), isDir:false, Bucket:"test-container", Name:"a/b/c/small.jpg", Size:19, ContentType:"", Created:time.Time{wall:0x0, ext:63761267128, loc:(*time.Location)(0x1039880)}, HashSum:"", Owner:(*owner.ID)(0xc00013ed98), Headers:map[string]string{}}
                            
                                Diff:
                                --- Expected
                                +++ Actual
                                @@ -8,3 +8,3 @@
                                  Size: (int64) 19,
                                - ContentType: (string) (len=25) "text/plain; charset=utf-8",
                                + ContentType: (string) "",
                                  Created: (time.Time) {
                Test:           Test_objectInfoFromMeta/a/b/c/small.jpg#01

```

--- a/api/layer/util.go
+++ b/api/layer/util.go
@@ -127,6 +127,7 @@ func objectInfoFromMeta(bkt *BucketInfo, meta *object.Object, prefix, delimiter 
 		index := strings.Index(tail, delimiter)
 		if index >= 0 {
 			isDir = true
+			mimeType = ""
 			filename = prefix + tail[:index+1]
 			userHeaders = nil
 		} else {

--- a/api/layer/util_test.go
+++ b/api/layer/util_test.go
@@ -28,12 +28,16 @@ func newTestObject(oid *object.ID, bkt *BucketInfo, name string) *object.Object 
 	created.SetKey(object.AttributeTimestamp)
 	created.SetValue(strconv.FormatInt(defaultTestCreated.Unix(), 10))
 
+	contentType := object.NewAttribute()
+	contentType.SetKey(object.AttributeContentType)
+	contentType.SetValue(defaultTestContentType)
+
 	raw := object.NewRaw()
 	raw.SetID(oid)
 	raw.SetOwnerID(bkt.Owner)
 	raw.SetContainerID(bkt.CID)
 	raw.SetPayload(defaultTestPayload)
-	raw.SetAttributes(filename, created)
+	raw.SetAttributes(filename, created, contentType)
 	raw.SetPayloadSize(uint64(defaultTestPayloadLength))
 
 	return raw.Object()

